### DISCUSSION TITLE
Redis Sentinel - RedisJSON Support

### DISCUSSION THREAD

**Discussion by crn717** [xetorthio/jedis#3252] [Q&A]
Hi,

I have a 3-node Redis environment(one master and two replicas) with Sentinel configured on each node.  In my application I use redis.clients.jedis.JedisSentinelPool to communicate with this Redis environment.  

I recently added the RedisJSON module to this environment.  It doesn't look like it's possible to use RedisJSON commands(e.g. json.set) via the redis.clients.jedis.Jedis instance that is obtained from redis.clients.jedis.JedisSentinelPool.  

Does the Jedis client library support RedisJSON in a Redis Sentinel environment?

Thanks,
Chris

**Comment by sazzad16** [comment]
Check JedisSentineled, available since Jedis 4.4.0-m1
